docs/deploy/gcp.md

```diff
@@ -59,21 +59,71 @@ For example, if you install lakeFS on GKE, you need to deploy the SQL Auth Proxy
    **Note:** it is preferable to run the binary as a service using systemd or your operating system's facilities.
 
 ### On Google Cloud Run
-To support container-based environments like Google Cloud Run, lakeFS can be configured using environment variables. Here is a `docker run`
-command to demonstrate starting lakeFS using Docker:
-
-```sh
-docker run \
-  --name lakefs \
-  -p 8000:8000 \
-  -e LAKEFS_DATABASE_TYPE="postgres" \
-  -e LAKEFS_DATABASE_POSTGRES_CONNECTION_STRING="[DATABASE_CONNECTION_STRING]" \
-  -e LAKEFS_AUTH_ENCRYPT_SECRET_KEY="[ENCRYPTION_SECRET_KEY]" \
-  -e LAKEFS_BLOCKSTORE_TYPE="gs" \
-  treeverse/lakefs:latest run
-```
-
-See the [reference](../reference/configuration.md#using-environment-variables) for a complete list of environment variables.
+To support container-based environments like Google Cloud Run, lakeFS can be configured using environment variables.
+
+1. Save the following configuration file as `config.yaml`:
+
+   ```yaml
+   apiVersion: serving.knative.dev/v1
+   kind: Service
+   metadata:
+     name: [LAKEFS_SERVICE_NAME]
+   spec:
+     template:
+       metadata:
+         annotations:
+           autoscaling.knative.dev/maxScale: '5'
+           run.googleapis.com/client-name: cloud-console
+           run.googleapis.com/cloudsql-instances: [POSTGRES_INSTANCE_NAME]
+       spec:
+         containers:
+         - image: gcr.io/lakefs/treeverse/lakefs@sha256:18be2f7e86f7572160909b7dcae2e50be5704cb174eeaf4fb61274def3536009
```

Comment:
This is internal repository in our cloud - we need to address using lakeFS under Google Cloud Run as it will not accept the one we use from docker hub.
So this one will not work for the end-user - unless they will push our image to their repository and use their address here.

Reply:
@nopcoder What instructions do customers need to push lakeFS image to their repository and use their address here? I reviewed GCP Container Registry. I can view our images but can't create new image to figure this out.

Can we make our image public? I saw this info under Settings in Container Registry UI:
"You can make the images under a Container Registry host accessible to the public. This allows users to pull images without authentication. The project will still be charged for Cloud Storage network usage."

Reply:
Our docker image is public and published under DockerHub. But Google Cloud Run requires to use GCP Container Registry. I pushed one of the lakeFS docker images manually to the GCP registry in order to use it. It is not a valid solution to make the specific version public - we need to have a release process to publish to such image or instruct to use how to push one of the current images to google. (first option is preferred).